tic_tac_toe/game.py

```diff
@@ -87,18 +141,54 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
+    board = game['board']
+    if _board_is_full(board) or game['winner']:
+        raise InvalidMovement("Game is over.")
+    if get_next_turn(game) == player:
```

Comment:
If you change these if statements around to not use else's and just check to see if an exception should be raised it would be a lot more legible.
Example:
```
if get_next_turn(game) != player:
    raise InvalidMovement('"{0}" moves next.'.format(game['next_turn']))
```
is much easier to understand than
```
if get_next_turn(game) == player:
    if other stuff:
        if more stuff:
            if yet more stuff:
                do some stuff
                do some other stuff
            else:
               raise Exception
        else:
            raise Exception
    else:
         raise Exception
else:
    raise InvalidMovement('"{0}" moves next.'.format(game['next_turn']))
```